features/collection/components/FilterableProductSection.tsx

```diff
@@ -46,6 +46,7 @@ export const FilterableProductSection = React.memo(() => {
    const productsContainerScrollRef = useScrollIntoViewEffect([hits]);
 
    if (collectionState === CollectionState.Empty) {
+      console.log('EMPTY!!');
```

Comment:
Do you still want this log? Seems like you don't...

Reply:
@cdcline I'll remove it 😅

Reply:
@cdcline fixed 👍